Formula/ice.rb

```diff
@@ -58,6 +58,7 @@ def install
     args = %W[
       prefix=#{prefix}
       USR_DIR_INSTALL=yes
+      SLICE_DIR_SYMLINK=yes
```

Comment:
Unfortunately no, not for a patch release. User applications and user build systems would rely on this location. We would be happy to change this behaviour for the next major release, 3.7.